ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-upi-e2e.yaml

```diff
@@ -1248,30 +1249,30 @@ objects:
+          ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-load-balancers.yaml" --list-tasks
+          ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-compute-nodes.yaml" --list-tasks
+          ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-control-plane.yaml" --list-tasks
+          ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-bootstrap.yaml" --list-tasks
+          ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-network.yaml" --list-tasks
+          ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-security-groups.yaml" --list-tasks
 
           # down-containers.yaml was introduced in 4.6
-          if [ -f "${OS_UPI_DIR}/down-containers.yaml" ]; then
-                ansible-playbook -vvv -i "${OS_UPI_DIR}/inventory.yaml" "${OS_UPI_DIR}/down-containers.yaml" --list-tasks
+          if [ -f "${ASSETS_DIR}/down-containers.yaml" ]; then
+                ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-containers.yaml" --list-tasks
           fi
         }
 
         function run_teardown_playbooks() {
-          ansible-playbook -vvv -i "${OS_UPI_DIR}/inventory.yaml" "${OS_UPI_DIR}/down-load-balancers.yaml"
-          ansible-playbook -vvv -i "${OS_UPI_DIR}/inventory.yaml" "${OS_UPI_DIR}/down-compute-nodes.yaml"
-          ansible-playbook -vvv -i "${OS_UPI_DIR}/inventory.yaml" "${OS_UPI_DIR}/down-control-plane.yaml"
-          ansible-playbook -vvv -i "${OS_UPI_DIR}/inventory.yaml" "${OS_UPI_DIR}/down-bootstrap.yaml"
-          ansible-playbook -vvv -i "${OS_UPI_DIR}/inventory.yaml" "${OS_UPI_DIR}/down-network.yaml"
-          ansible-playbook -vvv -i "${OS_UPI_DIR}/inventory.yaml" "${OS_UPI_DIR}/down-security-groups.yaml"
+          ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-load-balancers.yaml"
+          ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-compute-nodes.yaml"
+          ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-control-plane.yaml"
+          ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-bootstrap.yaml"
+          ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-network.yaml"
+          ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-security-groups.yaml"
 
           # down-containers.yaml was introduced in 4.6
-          if [ -f "${OS_UPI_DIR}/down-containers.yaml" ]; then
-                ansible-playbook -vvv -i "${OS_UPI_DIR}/inventory.yaml" "${OS_UPI_DIR}/down-containers.yaml" --list-tasks
+          if [ -f "${ASSETS_DIR}/down-containers.yaml" ]; then
+                ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-containers.yaml" --list-tasks
```

Comment:
ditto

Reply:
this line is changed in the next commit; it should be already like you suggest in the final version